src/package/sign/sign.proj

```diff
@@ -233,7 +233,7 @@
       <AssembliesToSign Include="$(ArtifactsDirectory)Extensions\%(ResxLang.Identity)\*.*" />
 
       <!-- Third Party assemblies to sign -->
-      <ThirdPartyAssembliesToSign Include="$(ArtifactsDirectory)Newtonsoft.Json.dll" />
+      <!--<ThirdPartyAssembliesToSign Include="$(ArtifactsDirectory)Newtonsoft.Json.dll" />-->
```

Comment:
Even though this assembly is already signed with the Newtonsoft certificate, we need to sign it again with the Microsoft 3rdParty certificate. Make sure we're doing that.

Reply:
We do sign it. This is duplicate of the same dll signing. You can find the same in 178 line.